core/src/test/scala/org/apache/spark/util/JsonProtocolSuite.scala

```diff
@@ -1097,7 +1100,9 @@ private[spark] object JsonProtocolSuite extends Assertions {
       |  },
       |  "Task Metrics": {
       |    "Executor Deserialize Time": 300,
+      |    "Executor Deserialize CPU Time": 0,
```

Comment:
AFAIK, JsonProtolSuite creates a JSON string from the event created by `makeTaskMetrics()`:
'makeTaskMetrics(300L, 400L, 500L, 600L, 700, 800, hasHadoopInput = true, hasOutput = false))'.
I tried changing `makeTaskMetrics()` to accept deserialize CPU time and CPU time as arguments , but that ended up violating scalaStyle by having more than 10 parameters..